packages/odyssey-react/src/components/Text/Text.test.tsx

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { render } from "@testing-library/react";
+import Text from ".";
+
+const children = "Text child content.";
+
+describe("Text", () => {
+  it("renders into the document", () => {
+    const { getByText } = render(<Text children={children} />);
+
+    expect(getByText(children)).toBeInTheDocument();
+  });
+
+  a11yCheck(() => render(<Text children={children} />));
+});
```

Comment:
I'd like to see more tests here to exercise the API and our types. That way we know the types are helpful to downstream users and will raise useful errors at compile time.